docker/templates/Dockerfile.test-database.m4

```diff
@@ -22,6 +22,7 @@ COPY \
     ./
 
 ENV PERL_CPANM_OPT --notest --no-interactive
+ENV test=true
```

Comment:
This should be removed from the Dockerfile (it should default to not set), and you should use a more descriptive name elsewhere, like `MB_IMPORT_DUMPS`.